File: Day14_Linear Search.cpp
#include <iostream>
using namespace std;

int linearSearch(int size, int num, int array[]){
	for(int i=0; i<size; i++){
		if(array[i] == num){
			cout<<"The Number "<<num<<" has been found at index "<<i;
			break;
		}
	}
	return 0;
}

int main(){
	
	int size;
	cout<<"Enter size of an array: ";
	cin>>size;
	int num;
	cout<<"Enter number you want to find: ";
	cin>>num;
	int arr[size];
	cout<<"Enter elements: ";
	for(int j=0; j<size; j++){
		cin>>arr[j];
	}
	
//	function call linearSearch
	linearSearch(size, num, arr);
	
	
	return 0;
}

File: Day14_Linear Search.py
def LinearSearch(array, num):
    for i in range(len(array)):
        if array[i] == num:
            return f"The searched number has been found at index {i}"

    return "The searched number was not found in the array"

# Input
size = int(input("Enter the size of the array: "))
number = int(input("Enter a number to search: "))
array = list(map(int, input(f"Enter an array of size {size} separated by spaces: ").split()))

# This is the function call for Linear Search
result = LinearSearch(array, number)
print(result)
